index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="./css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <nav class="my-nav">
+      <div class="logo">
+        <h2>Claudia</h2>
+      </div>
+      <div>
+        <div class="menu-toggle">&#9776;</div>
+      </div>
+    </nav>
+    <main class="container">
+      <h1>
+        I’m Claudia.
+        <br />
+        Glad to see
+        <br />
+        you!
```

Comment:
- [ ] Great job you really did a great work on this project but there are some little ammendment you just need to do please .
- [ ] The 2 sections you need to build for this project are:the toolbar (or header): and the the headline section (right after the header): which means you have to make it accessible and semantic by creating the headline section or main section right after the header please kindly fix that we know you can fix that we pround of you. to be more clear to you e.g for the Toolbar The container of the section is a header tag and  Headline The container of the section is a `section` tag.It represents a generic standalone section of the document
- [ ]  Your website should look exactly like the design template you choose on figma the design should look exact copy of Figma(e.g. font, colors, images, text, margins)
- [ ] Figma design 
![snapshot_www figma com_1682577269259](https://user-images.githubusercontent.com/92458236/234790111-2f3c45f6-bb60-44f0-8ca4-723b7e957ebf.png)

- [ ] Your design , please kindly add all the important background images to make it look exactly like the original template on figma.
![student template](https://user-images.githubusercontent.com/92458236/234790405-6eca550e-1b54-4189-8a54-e70ffa415b5a.png)